### ISSUE TITLE
[Project 1 - Review] Furry World website (landing page)

### ISSUE THREAD

**Issue by skadewdle** [tanaypratap/teamtanay.jobchallenge.dev#625]
Hi mentors!
I built this [landing page](https://furryworld.netlify.app) for my fathers business. HTML, CSS, Stylus (CSS) and Firebase were used to develop it. 

Please review it and suggest me improvements.

Thanks a lot!

**Comment by jasbindar-singh** [comment]
@skadewdle The landing page looks good
What I would like to suggest is,
1. Use smaller font-size, the one right now looks pretty big
2. Gallery has nothing in it
3. The site's layout breaks when using different screen sizes, work on responsiveness.
4. You could add more content, the content right now is very less.
5. Try to fill up some empty space.
And I want to know where are you using Firebase?

**Comment by skadewdle** [comment]
Yes, I'm still working on the gallery. I am using firebase storage for it, it is still in development. Thanks for the suggestions though. 🙂️
